CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 2.29.0-rc.0 / 2021-07-29
+
+* [CHANGE] Remove experimental tag on `--storage.tsdb.allow-overlapping-blocks` flag. #9117
+* [CHANGE] Remove experimental tag on `--storage.tsdb.retention.size` flag. #8728
```

Comment:
I guess we usually point to the PRs that made the change?
```suggestion
* [CHANGE] Remove experimental tag on `--storage.tsdb.retention.size` flag. #9004
```

Reply:
+1